File: fusionglpi/database/linked_vlan.py
from fusionglpi import db


class LinkedVlan(db.Model):
    id = db.Column(db.Integer, primary_key=True)
    vlan_id = db.Column(db.Integer, nullable=False, server_default="0")
    ip_network_id = db.Column(db.Integer, nullable=False, server_default="0")

File: app/dbmodels/LinkedVlan.php
<?php

$table = array(
    'fields'        => array(),
    'oldfields'     => array(),
    'renamefields'  => array(),
    'indexes'       => array(),
    'oldindexes'    => array(),
    'relationships' => array(),
    'model'         => 'LinkedVlan',
    'menu'          => ''
);

$table['fields'] = array(
    'id'               => DBModels::type('increments', 
                                         array('visible' => false)),
    'vlan_id'          => DBModels::type('integer'),
    'ip_network_id'  => DBModels::type('integer'),
);

